params/superchain.go

```diff
@@ -120,11 +120,7 @@ func LoadSuperChainConfig(opStackChainCfg *superchain.ChainConfig) *chain.Config
 		TerminalTotalDifficultyPassed: true,
 		Ethash:                        nil,
 		Clique:                        nil,
-		Optimism: &chain.OptimismConfig{
-			EIP1559Elasticity:        6,
-			EIP1559Denominator:       50,
-			EIP1559DenominatorCanyon: 250,
-		},
+		Optimism:                      nil,
```

Comment:
why don't you leave default value? optimism config will be set if `chConfig.Optimism != nil`. it means it will be nil in some case